src/ambianic/pipeline/store.py

```diff
@@ -174,16 +177,35 @@ def notify(self, save_json: dict):
         if self.notification is None:
             return
         log.debug("Sending notification(s)..")
+        directory = "{0}/ambianic-edge/premium.yaml".format(os.getcwd())
+
         # TODO extract inference data
         for inference_result in save_json['inference_result']:
-            data = {
-                'id': save_json['id'],
-                'label': inference_result['label'],
-                'confidence': inference_result['confidence'],
-                'datetime': save_json['datetime'],
-            }
-            notification = Notification(data=data, providers=self.notification_config["providers"])
-            self.notification.send(notification)
+            try: 
+                with open(directory) as file:
+                    file_data = yaml.load(file, Loader=yaml.FullLoader)
+                    credentials = file_data['credentials']
+
+                    header = {
+                        'Authorization': 'Bearer {0}'.format(
+                            credentials['ACCESS_TOKEN'])}
+
+                    userRequest = get(
+                        '{0}/userinfo'.format(credentials['AUTH_DOMAIN']), header)
+                    userData = userRequest.json()
+                    # TODO make a request to the serverless function using user's data
```

Comment:
Do we need to implement in code the call out to a premium rest notification service? Can we re-use the already available apprise plugin with the configuration settings from config.yaml for the specific service?
The PWA can ask the user for the notification settings (email subject, whether to include an attachment, etc) push these via the /config edge api. Then the notification service will do its job. That's already implemented.

In other words, I am not sure we need to modify `store.py`.

Reply:
Okay. 

I am looking into using Aprise notfication service